cg/server/api.py

```diff
@@ -467,6 +468,18 @@ def get_application_pipeline_limitations(tag: str):
     return jsonify([limitation.to_dict() for limitation in application_limitations])
 
 
+@BLUEPRINT.route("/orders")
+def get_orders():
+    """Return the latest orders."""
+    if request.args.get("limit"):
+        orders: list[Order] = db.get_orders(request.args.get(key="limit", type=int))
+    else:
```

Comment:
Don't call database logic directly in the endpoint and don't parse request arguments like this. Follow the [pattern in trailblazer](https://github.com/Clinical-Genomics/trailblazer/blob/0607d626f8e85e07398113db0fd50c4845db1d52/trailblazer/server/api.py#L62).

The pattern is
- Parse request body into pydantic model.
- Send pydantic model into service layer.
- Send data to database layer, which sends up database model to service layer.
- Service layer converts database model to pydantic response model.
- API layer dumps pydantic model into json response.

This can be refactored later, but if you want to practice writing clean endpoints it is a much better pattern.

In this case, you would just create a service called `OrderService` which has a single method `get_orders` which does not accept any arguments. Unless you want to implement pagination, then you can look at how it is done in trailblazer (I don't think it is worth it at this stage though).

Reply:
I like this pattern, do you think we can discuss/add it to the coding guidelines?

Reply:
Yeah, I will refactor this endpoint as well! Don't want to add to the technical debt!

Reply:
Refactored it a bit, although we have a slightly different method of parsing the request - do you see a reason for not using BaseModel.model_validate(request.args)?

Reply:
No, I think it is fine 👍 Might make more sense to do down the line